### DISCUSSION TITLE
How to use PGSync for both dev and test environments?

### DISCUSSION THREAD

**Discussion by rjmAmaro** [toluaina/pgsync#515] [Q&A]
Hello, I have a Rails project where I'm trying to integrate the usage of OpenSearch, using PGSync to sync data from PostgreSQL.

I want to test the features that are making requests to OpenSearch, but I need the data that is generated within the test (mock data) to be synced to OpenSearch with PGSync. This data is persisted at PostgreSQL on a dedicated table name `project_test`.

What is the best way to do this sync? Duplicate the schema inside `schema.json` to have two similar indexes, one for the dev database and one for the test database?

Thanks in advance, and a Happy New Year! 🎉

**Comment by toluaina** [comment]
I'm sorry for the delay. Yes duplicating the schema with different indices/databases pairs.
Alternatively, have 2 separate schemas and multiple instances of pgsync.

**Comment by gopal120399** [comment]
@rjmAmaro can you help please me with the integration of opensearch and PostgresSql via pgsync

Thank you
